tests/e2e/demo-components.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { isTesting } from '@lmc-eu/spirit-common/constants/environments';
+import { SERVERS, getDevelopmentEndpointUri } from '@lmc-eu/spirit-common/constants/servers';
+import { readdirSync } from 'fs';
+import { expect, test } from '@playwright/test';
+
+const runComponentTests = (testConfig) => {
+  const { packageDir, componentsDir, srcDir = '', hiddenComponents = [], type } = testConfig;
+  const name = type
+    .split('-')
+    .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
+    .join(' ');
+
+  test.describe(`Demo ${name} Components`, () => {
+    const componentDirs = readdirSync(`${packageDir}${srcDir}${componentsDir}`, { withFileTypes: true })
```

Comment:
This function is the same as in `app.ts`. Can we create one in the `common` package and use it?

Reply:
I tried, but I failed. I need it to work both in TS and outside TS and that caused problems. I suggest we fix it in a separate task.

Reply:
I agree. I've just reverted changes to the `common` package.